src/app/features/group/single-group/group-participants/group-participants.html

```diff
@@ -0,0 +1,18 @@
+<div *ngIf="errorClientServer$ | async; else success">
+  <app-error></app-error>
+</div>
+<ng-template #success>
+  <ng-container *ngIf="users$ | async as users">
+    <div class="main-container group">
+      <h1 class="font-heading-3">Członkowie grupy ({{users.groupMembers.length}})</h1>
+      <ul>
+        <div class="group__list">
+          <li class="font-text-1" *ngFor="let groupMember of users.groupMembers">
+            {{groupMember.firstName}} {{groupMember.lastName}}
+          </li>
+        </div>
+      </ul>
+    </div>
+  </ng-container>
+</ng-template>
+<app-cow-loader></app-cow-loader>
```

Comment:
Zgodnie ze znalezionym problemem z wyświetlaniem błedu,  
```
<div *ngIf="errorClientServer$ | async; else success">
  <app-error></app-error>
</div>
```
błąd przenieśmy na sam dół, a niech pierwszym będzie `<ng-container *ngIf="users$ | async as users">`
Wygłądało by to następująco 
```
<ng-container *ngIf="users$ | async as users; else error">
....
</ng-container>
<ng-template #error>
<div *ngIf="errorClientServer$ | async">
  <app-error></app-error>
</div>
</ng-template>